src/dstack/_internal/server/background/tasks/process_submitted_jobs.py

```diff
@@ -128,6 +128,15 @@ async def _process_submitted_job(session: AsyncSession, job_model: JobModel):
             instance = sorted_instances[0]
             instance.status = InstanceStatus.BUSY
             instance.job = job_model
+            logger.info(
+                "The job %s switched instance %s status to BUSY",
+                job_model.job_name,
+                instance.name,
+                extra={
+                    "instance_name": instance.name,
+                    "instance_status": InstanceStatus.BUSY.value,
+                },
+            )
```

Comment:
What's the choice for fields passed as extra? I suggest we pass all relevant context there. But we need to support extra for non-json logger as well – otherwise we'll duplicate the context in both message and extra.

Reply:
> What's the choice for fields passed as extra?

To see the lifecycle of an instance, it is easier to read when logs are formatted as a table. I added `instance_name` and `instance_status` as a columns to this "table". Each log with extra fields records a completed action or fact, not a context.